cli/config/configuration.toml.in

```diff
@@ -147,6 +147,10 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # Default false
 #hotplug_vfio_on_root_bus = true
 
+# If host doesn't support vhost_net, set to true. Thus we won't create vhost fds for nics.
+# Default false
+#disable_vhost_net = true
```

Comment:
So we no longger `modprobe` vhost-net, neither we fallback to non-vhost case, right? Please document it here that vhost-net kernel module needs to be builtin or inserted if vhost_net is enabled. Otherwise we'll fail to create the network endpoint for containers.

Reply:
It's OK to just fail in such case. Thanks for the info that vhost_net.ko is automatically loaded.